sim/mage/evocation.go

```diff
@@ -0,0 +1,89 @@
+package mage
+
+import (
+	"time"
+
+	"github.com/wowsims/sod/sim/core"
+)
+
+func (mage *Mage) registerEvocationSpell() {
+	actionID := core.ActionID{SpellID: 12051}
+	channelTime := time.Second * 8
+	cooldown := time.Minute * 8
+
+	tickLength := time.Millisecond * 250
+	maxTicks := int32(channelTime / tickLength)
+
+	manaRegenAura := mage.RegisterAura(core.Aura{
+		Label:    "Evocation Regen",
+		ActionID: actionID,
+		Duration: core.NeverExpires,
+		OnGain: func(aura *core.Aura, sim *core.Simulation) {
+			mage.PseudoStats.SpiritRegenMultiplier += 15
+			mage.PseudoStats.ForceFullSpiritRegen = true
+			mage.UpdateManaRegenRates()
+		},
+		OnExpire: func(aura *core.Aura, sim *core.Simulation) {
+			mage.PseudoStats.SpiritRegenRateCasting -= 1
```

Comment:
This should be  `-= 15`, right?

Reply:
Oh yeah I think this was a relic from when I was working on this. I think it should be
```
mage.PseudoStats.SpiritRegenMultiplier -= 15
```